themes/digital.gov/layouts/shortcodes/note.html

```diff
@@ -1,129 +1,49 @@
-
-Comment —
-{{< note "alert" >}}
-> Comment goes here as a blockquote additional text goes here...
-{{< /note >}}
-
-*/}}
-
-<div class="note {{ if .Get 0 }}{{- .Get 0 -}}{{ end }}">
-  {{- if and (.Get 0) (eq (.Get 0) "activity") -}}
-    <h4>
-      <svg
-        class="usa-icon dg-icon dg-icon--large margin-bottom-05"
-        aria-hidden="true"
-        focusable="false"
-      >
-        <use
-          xlink:href="{{ "uswds/img/sprite.svg#assessment" | relURL }}"
-        ></use>
-      </svg>
-      Activity
-    </h4>
-  {{- else if and (.Get 0) (eq (.Get 0) "action") -}}
-    <h4>
-      <svg
-        class="usa-icon dg-icon dg-icon--large margin-bottom-05"
-        aria-hidden="true"
-        focusable="false"
-      >
-        <use xlink:href="{{ "uswds/img/sprite.svg#campaign" | relURL }}"></use>
-      </svg>
-      Action
-    </h4>
-  {{- else if and (.Get 0) (eq (.Get 0) "alert") -}}
-    <h4>
-      <svg
-        class="usa-icon dg-icon dg-icon--large margin-bottom-05"
-        aria-hidden="true"
-        focusable="false"
-      >
-        <use xlink:href="{{ "uswds/img/sprite.svg#report" | relURL }}"></use>
-      </svg>
-      Alert
-    </h4>
-  {{- else if and (.Get 0) (eq (.Get 0) "comment") -}}
-    <h4>
-      <svg
-        class="usa-icon dg-icon dg-icon--large margin-bottom-05"
-        aria-hidden="true"
-        focusable="false"
-      >
-        <use xlink:href="{{ "uswds/img/sprite.svg#comment" | relURL }}"></use>
-      </svg>
-      Comment
-    </h4>
-  {{- else if and (.Get 0) (eq (.Get 0) "video") -}}
-    <h4>
-      <svg
-        class="usa-icon dg-icon dg-icon--large margin-bottom-05"
-        aria-hidden="true"
-        focusable="false"
-      >
-        <use
-          xlink:href="{{ "uswds/img/sprite.svg#fast_forward" | relURL }}"
-        ></use>
-      </svg>
-      Video
-    </h4>
-  {{- else if and (.Get 0) (eq (.Get 0) "group") -}}
-    <h4>
-      <svg
-        class="usa-icon dg-icon dg-icon--large margin-bottom-05"
-        aria-hidden="true"
-        focusable="false"
-      >
-        <use xlink:href="{{ "uswds/img/sprite.svg#groups" | relURL }}"></use>
-      </svg>
-      Who can join?
-    </h4>
-  {{- else if and (.Get 0) (eq (.Get 0) "plain") -}}
-  {{- else -}}
-    <h4>
-      <svg
-        class="usa-icon dg-icon dg-icon--large margin-bottom-05"
-        aria-hidden="true"
-        focusable="false"
-      >
-        <use
-          xlink:href="{{ "uswds/img/sprite.svg#notifications" | relURL }}"
-        ></use>
-      </svg>
+{{ $variant := (.Get "variant") }}
+{{ $icon := "notifications" }}
+
+{{ if eq $variant "activity" }}
+  {{ $icon = "assessment" }}
+{{ else if eq $variant "action" }}
+  {{ $icon = "campaign" }}
+{{ else if eq $variant "alert" }}
+  {{ $icon = "report" }}
+{{ else if eq $variant "comment" }}
+  {{ $icon = "comment" }}
+{{ else if eq $variant "video" }}
+  {{ $icon = "fast_forward" }}
+{{ else if eq $variant "Who can join?" }}
+  {{ $icon = "groups" }}
```

Comment:
Let's not use `Who to join` and prefer `join` for the variant. Updated existing variant to use this one.